minutes/2023-05-23.md

```diff
@@ -0,0 +1,27 @@
+# Parsec Community Meeting (23/05/2023) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Adam Johnston (@adamjohnstonarm)
+- Ionut Mihalcea (@ionut-arm)
+- Paul Howard (@paulhowardarm)
+- Gowtham Suresh Kumar (@gowthamsk-arm)
+- Vikas Katariya
```

Comment:
Do we have a github account that we can reference Vikas to?

Reply:
Vikas is https://github.com/jainvikas8